ts/components/ui/ActivityIndicator.tsx

```diff
@@ -1,17 +1,24 @@
 import * as React from "react";
-import { ActivityIndicator as RNActivityIndicator } from "react-native";
+import {
+  ActivityIndicator as RNActivityIndicator,
+  ActivityIndicatorProps
+} from "react-native";
 
 import variables from "../../theme/variables";
+import { WithTestID } from "../../types/WithTestID";
 
-interface Props {
+type Props = WithTestID<{
   color?: string;
-}
+  accessibilityLabel?: string;
+  size?: ActivityIndicatorProps["size"];
+}>;
```

Comment:
since `color`, `accessibilityLabel` and `color` are all included and possibly undefined as definition of `ActivityIndicatorProps` why not using directly default prop definition at least omitting unused specific props, such as `animating` and `hidesWhenStopped`.

`testID` should be also included

Reply:
☝🏼 ✅ 
I guess we can allow all props, to make the component more customizable
https://github.com/pagopa/io-app/pull/3229/commits/136d41148d69aa502c215e81a23a14d11120712a